src/app/components/stats/stats.component.html

```diff
@@ -0,0 +1,110 @@
+                ></div>
+                <div class="bg-violet-950 w-1/12 h-full"></div>
+              </div>
+              <div
+                class="bg-mantis-green flex flex-row justify-center shadow-inner shadow-gray-450 rounded-md w-3/5 h-1/4 gap-14"
+              >
+                <div class="bg-green-800 w-[8%] h-full"></div>
+                <div class="bg-green-800 w-[8%] h-full"></div>
+              </div>
+              <div
+                class="bg-pink-900 rounded-l-3xl ml-4 rounded-r-md flex justify-end items-center w-3/5 h-[30%]"
+              >
+                <div
+                  class="bg-linen w-19/20 h-13/20 shadow-inner rounded-l-xl shadow-gray-400 lg:rounded-l-2xl"
+                ></div>
+              </div>
+            </div>
+          </div>
+          <div
+            class="books-read-info-container h-full w-3/5 flex flex-col justify-center items-center"
+          >
+            <p class="text-white w-full text-center">
+              You have read a total of
+            </p>
+            <p class="text-white font-semibold text-4xl">20</p>
+            <p class="text-white">books.</p>
+          </div>
+        </div>
+        <div
+          class="w-1/3 h-full bg-gradient-to-r from-old-gold via-burnt-sienna to-amaranth rounded-r-3xl"
+        ></div>
+      </div>
+
+      <div
+        class="pages-read-container bg-amaranth rounded-2xl w-full h-[10%] flex flex-row justify-start gap-4 items-center pl-2"
+      >
+        <p class="bg-seasalt rounded-3xl w-fit h-fit px-2 py-1 text-2xl">📃</p>
+        <p class="text-white">You have read a total of 1000 pages</p>
+      </div>
+      <div
+        class="prefered-pace-container bg-mantis-green rounded-2xl w-full h-[10%] flex flex-row justify-start gap-4 items-center pl-2"
+      >
+        <p class="bg-seasalt rounded-3xl w-fit h-fit px-2 py-1 text-2xl">⚡️</p>
+        <p class="text-white">You prefer moderately-paced books.</p>
+      </div>
+      <div
+        class="preferred-genre-container bg-indigo-dye rounded-2xl w-full h-[10%] flex flex-row justify-start gap-4 items-center pl-2"
+      >
+        <p class="bg-seasalt rounded-3xl w-fit h-fit px-2 py-1 text-2xl">📚</p>
+        <p class="text-white">You prefer moderately-paced books.</p>
+      </div>
+      <div
+        class="prefered-mood-container bg-dull-violet rounded-2xl w-full h-[10%] flex flex-row justify-start gap-4 items-center pl-2"
+      >
+        <p class="bg-seasalt rounded-3xl w-fit h-fit px-2 py-1 text-2xl">😃</p>
+        <p class="text-white">You prefer funny books.</p>
+      </div>
+      <div
+        class="average-rating-container bg-old-gold rounded-2xl w-full h-[10%] flex flex-row justify-start gap-4 items-center pl-2"
+      >
+        <p class="bg-seasalt rounded-3xl w-fit h-fit px-2 py-1 text-2xl">⭐️</p>
+        <p class="text-white">Your average rating is 4 stars.</p>
+      </div>
+    </div>
+  </div>
```

Comment:
div city is lovely this time of year, but maybe add some labels/ids or aria-labels for reference

Reply:
Will look into this when I return to UI.